src/ICARECSVClient.js

```diff
@@ -18,6 +18,9 @@ class ICARECSVClient extends BaseClient {
       CSVPatientExtractor,
       CSVTreatmentPlanChangeExtractor,
     );
+
+    this.commonExtractorArgs = commonExtractorArgs;
+    this.commonExtractorArgs.implementation = commonExtractorArgs.implementation ? commonExtractorArgs.implementation : 'icare';
```

Comment:
DP - After making all my comments below about adding another condition to our conditional, I think there's a more elegant way of doing this, with something like the following: 
```
this.commonExtractorArgs = { 
    implementation: 'icare',
    ...commonExtractorArgs
}
```

This will always set the implementation variable to icare, but will overwrite it if there's a different value set by the incoming `commonExtractorArgs`

Reply:
DP - This addressed everything above flawlessly! Great suggestion :). I was able to wholly remove the `commonExtractorArgs` checks in both `cliUtils.js` and the `ICARECSVClient` test file.